Libraries/ReactNative/AppContainer.js

```diff
@@ -47,18 +44,6 @@ class AppContainer extends React.Component<Props, State> {
 
   static getDerivedStateFromError: any = undefined;
 
-  static childContextTypes:
```

Comment:
This would break what is available to children as part of the old context api. I think that would make this a breaking change forcing everyone to use the new RootTagContext. We can't make that breaking change yet at FB as we still have too many callsites using that. There is a side effort internally to get everything off the legacy context for RootTag though. This might be blocked until then.

Reply:
Could we make the change to Modal without making this change?

Reply:
Ok I'll add it back tomorrow

Reply:
Updated the PR